addons/frag/CfgAmmoFragParameters.hpp

```diff
@@ -0,0 +1,886 @@
+
+// ~~~~ Mines & UXO
+class ATMine_Range_Ammo: MineBase {
+    GVAR(skip) = 1;
+};
+
+class APERSMine_Range_Ammo: MineBase { // VS-50
+    class EventHandlers: EventHandlers {};
+    GVAR(skip) = 0;
+    GVAR(force) = 0;
+    GVAR(classes)[] = {QGVAR(tiny), QGVAR(tiny), QGVAR(small)};
+    GVAR(fragCount) = 500;
+    GVAR(metal) = 100;
+    GVAR(charge) = 50;
+    GVAR(gurney_c) = 2700;
+    GVAR(gurney_k) = "1/3";
+};
+
+class APERSBoundingMine_Range_Ammo: MineBase {
+    GVAR(skip) = 0;
+    GVAR(force) = 0;
+    GVAR(classes)[] = {QGVAR(tiny)};
+    GVAR(fragCount) = 200;
+    GVAR(metal) = 80;
+    GVAR(charge) = 170;
+    GVAR(gurney_c) = 2700;
+    GVAR(gurney_k) = "3/5";
+};
+
+class TrainingMine_Ammo: APERSMine_Range_Ammo {
+    GVAR(skip) = 1;
+};
+
+class SLAMDirectionalMine_Wire_Ammo: DirectionalBombBase {
+    GVAR(skip) = 1;
+};
+
+class APERSTripMine_Wire_Ammo: DirectionalBombBase {
+    GVAR(skip) = 0;
+    GVAR(force) = 0;
+    GVAR(classes)[] = {QGVAR(tiny)};
+    GVAR(fragCount) = 800;
+    GVAR(metal) = 210;
+    GVAR(charge) = 185;
+    GVAR(gurney_c) = 2843;
+    GVAR(gurney_k) = "3/5";
+};
+
+class IEDUrbanBig_Remote_Ammo: PipeBombBase {
+    GVAR(skip) = 0;
+    GVAR(classes)[] = {QGVAR(large), QGVAR(large), QGVAR(large_HD), QGVAR(large), QGVAR(huge), QGVAR(huge_HD), QGVAR(huge)};
+    GVAR(fragCount) = 2000;
+    GVAR(metal) = 36000;
+    GVAR(charge) = 9979;
+    GVAR(gurney_c) = 2440;
+    GVAR(gurney_k) = "3/5";
+};
+class IEDLandBig_Remote_Ammo: PipeBombBase {
+    GVAR(skip) = 0;
+    GVAR(classes)[] = {QGVAR(large), QGVAR(large), QGVAR(large_HD), QGVAR(large), QGVAR(huge), QGVAR(huge_HD), QGVAR(huge)};
+    GVAR(fragCount) = 2000;
+    GVAR(metal) = 36000;
+    GVAR(charge) = 9979;
+    GVAR(gurney_c) = 2440;
+    GVAR(gurney_k) = "3/5";
+};
+class IEDUrbanSmall_Remote_Ammo: PipeBombBase {
+    GVAR(skip) = 0;
+    GVAR(fragCount) = 1200;
+    GVAR(metal) = 23000;
+    GVAR(charge) = 3148;
+    GVAR(gurney_c) = 2830;
+    GVAR(gurney_k) = "3/5";
+    GVAR(classes)[] = {QGVAR(small), QGVAR(medium), QGVAR(medium_HD)};
+};
+class IEDLandSmall_Remote_Ammo: PipeBombBase {
+    GVAR(skip) = 0;
+    GVAR(fragCount) = 1200;
+    GVAR(metal) = 23000;
+    GVAR(charge) = 3148;
+    GVAR(gurney_c) = 2830;
+    GVAR(gurney_k) = "3/5";
+    GVAR(classes)[] = {QGVAR(small), QGVAR(medium), QGVAR(medium_HD)};
+};
+
+class APERSMineDispenser_Ammo: PipeBombBase {
+    GVAR(skip) = 1;
+};
+
+class UXO1_Ammo_Base_F: APERSMine_Range_Ammo { // Mk 118 Rockeye
+    GVAR(skip) = 0;
+    GVAR(charge) = 180;
+    GVAR(metal) = 400;
+    GVAR(gurney_c) = 2700;
+    GVAR(gurney_k) = "1/2";
+    GVAR(fragCount) = 200;
+    GVAR(classes)[] = {QGVAR(small)};
+};
+
+class UXO2_Ammo_Base_F: UXO1_Ammo_Base_F {
+    GVAR(charge) = 100;
+    GVAR(fragCount) = 304;
+    GVAR(classes)[] = {QGVAR(small), QGVAR(tiny)};
+};
+
+class APERSMineDispenser_Mine_Ammo: APERSMine_Range_Ammo {
+    GVAR(skip) = 1;
+};
+
+class UXO_deploy_base_f: SubmunitionBase;
```

Comment:
```suggestion
class UXO_deploy_base_f: SubmunitionBase {};
```

Reply:
Thanks. I had been just reading the reports and fixing them as they come up. It is pretty late and, it's pretty obvious I am making some dumb mistakes, but ignoring that is there any downside fucking up so many times in a row (wrt running checks) other than exposing your shame?

Reply:
I don't think so - and I hope not for my sake xD